flux/flux-system/kutomization-ngnix-ingress-controller.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: kustomize.toolkit.fluxcd.io/v1beta1
```

Comment:
What is this one for?

Reply:
this is for installing ingress controller using flux CD, I can keep this in separate PR maybe

Reply:
No need, all good 👍🏽